internal/replacements/replace_failed_process_groups.go

```diff
@@ -45,15 +45,26 @@ func getMaxReplacements(cluster *fdbv1beta2.FoundationDBCluster, maxReplacements
 
 // ReplaceFailedProcessGroups flags failed processes groups for removal and returns an indicator
 // of whether any processes were thus flagged.
-func ReplaceFailedProcessGroups(log logr.Logger, cluster *fdbv1beta2.FoundationDBCluster, status *fdbv1beta2.FoundationDBStatus, hasDesiredFaultTolerance bool) bool {
-	// Automatic replacements are disabled, so we don't have to check anything further
-	if !cluster.GetEnableAutomaticReplacements() {
-		return false
+func ReplaceFailedProcessGroups(log logr.Logger, cluster *fdbv1beta2.FoundationDBCluster, status *fdbv1beta2.FoundationDBStatus, hasDesiredFaultTolerance bool) (bool, bool) {
```

Comment:
I think named return variables could be really helpful here, or at least add a note in the docstring about the return order